build/gh-pages-app.js

```diff
@@ -156,6 +156,8 @@ class GhPagesApp {
       versionJs,
       ...this._resolveSourcesList(this.opts.javascripts),
     ];
+    if (process.env.DEBUG) return contents.join('\n');
```

Comment:
aye... i've been adding the undocumented `beautify: true` in the options which is almost equivalent.